gateway_test.go

```diff
@@ -129,6 +129,11 @@ func TestMain(m *testing.M) {
 	config.EnableJSVM = true
 	config.Monitor.EnableTriggerMonitors = true
 	config.AnalyticsConfig.NormaliseUrls.Enabled = true
+	config.SlaveOptions.CallTimeout = 30
+	config.SlaveOptions.PingTimeout = 60
+	GlobalRPCPingTimeout = time.Second * time.Duration(config.SlaveOptions.PingTimeout)
+	GlobalRPCCallTimeout = time.Second * time.Duration(config.SlaveOptions.CallTimeout)
```

Comment:
I'll be frank, this change does not look that good, especially we if we change config options, we will have to modify them there too. 

What about moving default config into a variable outside `writeDefaultConf` function: 
```
var defaultConf = Config{
 		ListenPort:     8080,
                ....
}
```

And then use this variable in `writeDefaultConf` and test initialization: `config = defaultConf`.